app/javascript/packs/billboard.js

```diff
@@ -0,0 +1,23 @@
+import { setupDisplayAdDropdown } from '../utilities/displayAdDropdown';
+
+// the term billboard can be synonymously interchanged with displayAd
+async function getBillboard() {
+  const placeholderElement = document.getElementsByClassName(
+    'js-display-ad-comments-container',
+  )[0];
+
+  const { asyncUrl } = placeholderElement.dataset || {};
+
+  if (placeholderElement.innerHTML.trim() === '') {
+    const response = await window.fetch(`${asyncUrl}`);
```

Comment:
In the next iteration I'll update this JavaScript to cater for more than one ad.